pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -577,10 +590,10 @@ func (adc *attachDetachController) nodeDelete(obj interface{}) {
 	nodeName := types.NodeName(node.Name)
 	if err := adc.desiredStateOfWorld.DeleteNode(nodeName); err != nil {
 		// This might happen during drain, but we still want it to appear in our logs
-		klog.Infof("error removing node %q from desired-state-of-world: %v", nodeName, err)
+		logger.Info("Error removing node from desired-state-of-world", "node", klog.KRef("", string(nodeName)), "err", err)
```

Comment:
```suggestion
		logger.Info("Error removing node from desired-state-of-world", "node", klog.KObj(node), "err", err)
```

Also applies below whenever there is a `node` variable.

Reply:
done